### DISCUSSION TITLE
Account has been flagged

### DISCUSSION THREAD

**Discussion by abdullahkhan044** [community/community#35039] [New to GitHub] [answered]
My account has been flagged why?
I just make an account 
What is the reason for this?
I do not have multiple accounts
kindly help me to solve this issue

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them as the Community is unable to assist in matters of flagged accounts :)
